doc/classes/WorkerThreadPool.xml

```diff
@@ -15,6 +76,9 @@
 			<param index="3" name="high_priority" type="bool" default="false" />
 			<param index="4" name="description" type="String" default="&quot;&quot;" />
 			<description>
+				Registers a [Callable] for execution on [param tasks_needed] threads. If [param tasks_needed] equals [code]-1[/code] (the default), the worker requests all computer processor cores.
+				[param elements] represents how often your [Callable] is executed. Each execution receives a next higher index (up to [code]elements - 1[/code]).
```

Comment:
This is very confusing if not misleading.
`how often your [Callable] is executed`? Five times per second? Ten times per second? It's rather `how many times` it's called, rather than `how often`.

The name of the parameter is also poorly chosen. `Elements` suggests that it's a collection, but in reality it is an integer. Wouldn't it be better as `counter` or something similar?

Reply:
1) indeed, "how many times" is correct. thanks!

2) that is out of scope of writing the class documentation. the parameter name is auto-pulled from the code and should be changed there first, if at all. the worker thread pool expects you to iterate through a collection when processing a group task, so the name kinda fits once you understand how it works.